src/Utils/identityVerificationRequirements.ts

```diff
@@ -25,8 +25,8 @@ export const bidderNeedsIdentityVerification = ({
   bidder,
 }: {
   sale: IdentityVerificationRequireable
-  user: IdentityVerifiable | null
-  bidder: Bidder | null
+  user?: IdentityVerifiable
+  bidder?: Bidder
 }) => {
   return (
     !bidder?.qualified_for_bidding &&
```

Comment:
I realize this is not part of the diff but why is it necessary to check if the bidder is qualified for bidding? Wouldn't that be encapsulated by the checks below?

Reply:
@erikdstock Check out [this slack thread](https://artsy.slack.com/archives/CA3D02SDN/p1586546440286600) and [this bug](https://artsyproduct.atlassian.net/browse/AUCT-985) – both should help clarify why this is needed.